src/MCPServer/tests/test_package.py

```diff
@@ -376,3 +377,42 @@ def test_move_file(self, tmp_path, processing_dir, transfer):
 
         transfer.refresh_from_db()
         assert Path(transfer.currentlocation) == dest_path
+
+
+@pytest.mark.parametrize(
+    "package_class,model_class",
+    [
+        (
+            Transfer,
+            models.Transfer,
+        ),
+        (
+            SIP,
+            models.SIP,
+        ),
+        (
+            DIP,
+            models.SIP,
+        ),
+    ],
+)
+@pytest.mark.django_db(transaction=True)
+def test_package_statuses(package_class, model_class):
+    package_id = uuid.uuid4()
+    model_class.objects.create(pk=package_id)
+    package = package_class("/tmp", package_id)
```

Comment:
I don't think the package classes do anything on their own with the `path` currently, but maybe use the `tmp_path` fixture just in case?

Reply:
Fixed! Thanks.